pkg/services/check/check.go

```diff
@@ -0,0 +1,110 @@
+package check
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/weaveworks/weave-gitops/pkg/flux"
+	"github.com/weaveworks/weave-gitops/pkg/kube"
+)
+
+var ErrFluxNotFound = errors.New("flux is not installed")
+var ErrKubernetesNotFound = errors.New("no kubernetes version found")
+
+func Pre(ctx context.Context, kubeClient kube.Kube, fluxClient flux.Flux, expectedFluxVersion string) (string, error) {
+	output := ""
+
+	k8sOutput, err := validateKubernetes(fluxClient)
+	if err != nil {
+		return "", err
+	}
+
+	output += k8sOutput
+
+	currentFluxVersion, err := getCurrentFluxVersion(ctx, kubeClient)
+	if err != nil {
+		if errors.Is(err, ErrFluxNotFound) {
+			output += "✔ Flux is not installed"
```

Comment:
@J-Thompson12 I'm confused, what do you mean?

Reply:
The check mark in the error text. "✔ Flux is not installed". It seems odd to have a check mark for a fail.

Reply:
@J-Thompson12  Well, the purpose of this new command is to know if a cluster is ready to install gitops on it. Flux not installed is a good scenario, because we are not expecting flux to be installed (at least not yet, there are tickets already defined for that)